pkg/crc/machine/vsock.go

```diff
@@ -12,11 +12,9 @@ import (
 )
 
 func exposePorts(preset crcPreset.Preset) error {
-	daemonClient := daemonclient.New()
 	portsToExpose := vsockPorts(preset)
-	alreadyOpenedPorts, err := daemonClient.NetworkClient.List()
+	daemonClient, alreadyOpenedPorts, err := getClientAndOpenPorts()
```

Comment:
functions returning multiple values, and listing them all in their names just confuse me :) Maybe this can be:
```
daemonClient := daemonclient.New()
alreadyOpenedPorts, err := listOpenPorts(daemonClient)
```
?